### PR TITLE
initial vcash template

### PR DESCRIPTION
**Pull request by john-connor**
initial vcash template

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@john-connor__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@john-connor__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: vcash-on-ubuntu/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Vcash Node on Ubuntu VM
+
+This Microsoft Azure template deploys a Vcash public node that will connect to the main Vcash decentralised network.
```
Comment (singhkay) [inline vcash-on-ubuntu/README.md]: Need the deploy to azure and visualize buttons

Comment (singhkay) [inline vcash-on-ubuntu/README.md]: Should add more information about what Vcash is and links to Github repo, project site, documentation

Comment (john-connor) [inline vcash-on-ubuntu/README.md]: Done

**Comment by singhkay** [comment]
CI passed. lgtm, merged

**Comment by john-connor** [comment]
Thanks. We are moving on to PPA install next.

**Pull request merged**